February 17 release notes (#729)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/changelog.mdx

    >
      The Commerce boilerplate has been updated with incremental enhancements building on the January 2026 release:

      **Breaking changes:**

      - **Node.js 24 required** - The boilerplate now requires Node.js 24. Update your development environment before upgrading.
      - **Cards List block removed** - The `cards-list` block has been removed. Use the `cards` block instead.
      - **Fragment block behavior** - Fragment blocks now replace their content directly instead of replacing the entire section wrapper, which may affect existing fragment implementations.

      **Document Authoring improvements:**

      - **Universal Editor model v2** - Upgraded component definitions to model v2 for improved authoring experience ([#1086](https://github.com/hlxsites/aem-boilerplate-commerce/pull/1086)).
      - **Commerce block definitions** - Added comprehensive JSON definitions for all commerce blocks including Account Header, Account Sidebar, Addresses, Cart, Checkout, Mini Cart, Orders List, Wishlist, and more, enabling visual authoring in Universal Editor and Document Authoring ([#1086](https://github.com/hlxsites/aem-boilerplate-commerce/pull/1086)).

>       - **Improved field naming** - Block field definitions now use semantic names (e.g., `image`, `text`, `summary`) instead of CSS selector-based names ([#1086](https://github.com/hlxsites/aem-boilerplate-commerce/pull/1086)).

Reviewer: ```suggestion
      - **Improved field naming** - Block field definitions now use semantic names (such as `image`, `text`, `summary`) instead of CSS selector-based names ([#1086](https://github.com/hlxsites/aem-boilerplate-commerce/pull/1086)).
```

**Drop-in SDK** 
*****************/}

    <ChangelogEntry
      date="2026-02-17"
      title="Drop-in SDK (Elsie) v1.7.0"
      components={['Drop-in SDK']}
    >
      The Drop-in SDK (Elsie) has been updated with the following changes:

>       - **Revert: field asterisk styling** - Reverted wrapping required field asterisks in span elements for consistent styling ([#161](https://github.com/adobe-commerce/StorefrontSDK/pull/161)).

Reviewer 2: Could you please remove this line? this change was reverted in the release branch (it's not included into the previous stable version)

      - **Custom content below row total** - Allow custom content below cart item row total price ([#162](https://github.com/adobe-commerce/StorefrontSDK/pull/162)).
      - **Price display fix** - Resolved issue with displaying prices.
      - **RadioButton icon support** - Added icon support to the RadioButton component.

>       - **Cart item image border fix** - Fixed border around cart item's image when inside a link.

Reviewer 2: Could you please remove this line? this change was reverted in the release branch (it's not included into the previous stable version)

    >
      The Storefront Compatibility Package has been updated to include the following changes:

>       - Added the **Merge Cart Preference** field to the **Store** > **Sales** > **Checkout** > **Shopping Cart** configuration panel in the Admin. It controls cart merge behavior when a GraphQL mutation merges guest or logged-in customer carts.

Reviewer: This field needs to be added to the Admin Config Reference.

Author: I need to understand this better. I'll add this in another PR since it was in the 2025-06-25 release.

**Order** 
*********/}

    <ChangelogEntry
      date="2026-02-17"
      title="Order v3.1.0"
      components={['Order']}
    >
      The Order drop-in has been updated with the following changes:

>       - **Gift wrapping availability** - Added `gift_wrapping_available` field to order item product details, enabling conditional display of gift wrapping options based on product configuration ([#65](https://github.com/adobe-commerce/storefront-order/pull/65)).

Reviewer: I think you need to add giftWrappingAvailable to the order details data model doc

**Product details page** 
************************/}

    <ChangelogEntry
      date="2026-02-17"
      title="Product details page v3.1.0"
      components={['Product details page']}
    >
      The Product Details Page drop-in has been updated with the following changes:

      - **Downloadable products support** - Added support for downloadable product options directly on the PDP, allowing customers to view and select downloadable links and samples ([#50](https://github.com/adobe-commerce/storefront-pdp/pull/50)).

>       - **Options transformer prop** - New `transformer` prop enables custom option type conversion with robust error handling for advanced product configuration scenarios.

Reviewer: not sure where to check if this has been doc'd. It's not in dropins/product-details/containers/product-options/, but I don't know if I'm looking in the right place

Author: Good catch. You were looking in the right drop-in but on the wrong page. The transform and transformer props live on the PDP initializer's models configuration, not on the ProductOptions container. 

A search for them revealed they were not documented.

**User Account** 
****************/}

    <ChangelogEntry
      date="2026-02-17"
      title="User Account v3.1.0"
      components={['User Account']}
    >
      The User Account drop-in has been updated with the following changes:

      - **Dynamic form field translations** - Added dynamic i18n support for field labels and placeholders, enabling custom placeholders and floating labels in address forms ([#40](https://github.com/adobe-commerce/storefront-account/pull/40)).

>       - **Improved accessibility** - New `getAriaLabel` method for better screen reader support in form elements.

Reviewer: Not doc'd in dropins/user-account/functions/ (Same caveat applies)

Author: Changed bullet to remove the `getAriaLabel` because the implementation is internal and automatically applied.

**Quote Management** 
********************/}

    <ChangelogEntry
      date="2026-02-17"
      title="Quote Management v1.1.0"
      components={['Quote Management']}
    >
      The Quote Management drop-in has been updated with the following changes:

>       - **Quote template expiration dates** - New `setQuoteTemplateExpirationDate` API enables managing expiration dates for quote templates, with a new modal component for setting dates through a user-friendly interface ([#78](https://github.com/adobe-commerce/storefront-quote-management/pull/78)).

Reviewer: Not doc'd in dropins-b2b/quote-management/functions/, but that's an AI generated file, so maybe it's forthcoming.

Author: I added it manually for now.
